.gitignore

```diff
@@ -8,3 +8,6 @@ Rhai.toml
 **/*.bat
 doc/rhai-sync.json
 doc/rhai.json
+.idea/
```

Comment:
Probably should remove `.idea` here as they don't exist in the official repo.

Reply:
Those are intellij idea files, they are there for the same reason `.vscode` is there - IDE specific files. I think its a good idea to keep those in case someone else also uses intellij-rust (me)